src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -71,12 +71,14 @@ private static boolean commonHelp(String mode, boolean canConnectToRemote) {
             System.out.println("    --connect [<serverid>@]<host>[:registryport][/servername] To connect to a remote debug server (debugd).");
         }
         System.out.println();
+        if (canConnectToRemote) {
+            System.out.println("    The --connect option is deprecated and will be removed in a future release.");
```

Comment:
It is ok to add this, but now there is no longer any help message to explain what `--connect` does.

Reply:
We have line 71 saying "--connect .....To connect to a remote debug server (debugd)."
It's a long line and scanning it people might not read it all, but too late to worry overly about that. 8-)

Reply:
Ok. I was looking at the deletion of line 79. I didn't realize it was removed because it pretty much just repeated line 71.